CONTRIBUTING.md

```diff
@@ -74,6 +74,31 @@ We expect development to follow a typical "forking" style workflow:
 4. PRs are associated with [Github Issues](https://github.com/microsoft/MLOS/issues) and need [MLOS-committers](https://github.com/orgs/microsoft/teams/MLOS-committers) to sign-off (in addition to other CI pipeline checks like tests and lint checks to pass).
 5. Once approved, the PR can be completed using a squash merge in order to keep a nice linear history.
 
+## Distributing
```

Comment:
```suggestion
### Distributing
```
or did you mean it?

Reply:
No, I think those are slightly separate topics.